### PR TITLE
Add auto-update feature in netbird script for binary installation

### PR DESCRIPTION
**Pull request by bcmmbaga**
## Describe your changes

This pull request addresses the need to enhance the installer script by introducing a new parameter `--update` to trigger updates. The goal is to streamline the update process for binary installations and provide a better experience for users.

### Checklist
- [ ] Is it a bug fix
- [ ] Is a typo/documentation fix
- [x] Is a feature enhancement
- [ ] It is a refactor
- [ ] Created tests that fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: release_files/install.sh [hunk 1]
```diff
@@ -303,4 +303,43 @@ install_netbird() {
...
+
+      sudo netbird service stop
+      sudo netbird service uninstall
+      install_native_binaries
+
+      sudo netbird service install
+      sudo netbird service start
+    fi
+  else
+     echo "Netbird installation was done using a package manager. please use package to update"
```
Comment (braginini) [inline release_files/install.sh]:
```suggestion
     echo "Netbird installation was done using a package manager. Please use your system's package manager to update"
```

**Review by braginini — APPROVED** [review]

**Comment by Aqua1ung** [comment]
So, for _binary installations_, how does one _update_ NetBird? Just run `install.sh` again?

**Comment by bcmmbaga** [comment]
> So, for _binary installations_, how does one _update_ NetBird? Will running `curl -fsSL https://pkgs.netbird.io/install.sh | sh` again do it?

You can update by running `curl -fsSL https://pkgs.netbird.io/install.sh | sh -s -- --update`

**Comment by Aqua1ung** [comment]
> You can update by running `curl -fsSL https://pkgs.netbird.io/install.sh | sh -s -- --update`

I am guessing this an intelligent update (meaning no update if none needed), right?

**Comment by bcmmbaga** [comment]
> Thanks. I am guessing this an _intelligent_ update (meaning no update if none needed), right?

Correct. In case you are running the latest version, there will be no updates.

**Pull request merged**